### DISCUSSION TITLE
build error while trying to package it for void linux

### DISCUSSION THREAD

**Discussion by aryalaadi** [zed-industries/zed#8779] [Feature Requests]
https://github.com/void-linux/void-packages/actions/runs/8131780165/job/22221498916?pr=49019
cant seem to figure out whats happening, it builds perfectly fine when directly running `cargo build`

**Comment by Moshyfawn** [comment]
You may need to initialise submodules.

```bash
git submodule update --init --recursive
```

**Reply by aryalaadi** [reply]
void maintainers do not allow this, is it possible to provide "hydrated" sources through github releases? 
right now my approach is downloading the tar provided through zed github releases, and download the livekit sources as a way to "hack" in submodules
but some upstream maintainers provide a source archive with all the submodules initialized
